cirq-google/cirq_google/engine/engine_job.py

```diff
@@ -81,8 +79,7 @@ def __init__(
             job_id: Unique ID of the job within the parent program.
             context: Engine configuration and context to use.
             _job: The optional current job state.
-            result_type: What type of results are expected, such as
-                batched results or the result of a focused calibration.
+            result_type: What type of results are expected.
```

Comment:
Should we deprecate/remove this parameter since it must always be a `ResultType.Program`? If it's not removed.

Also - should we remove the `Batch` and `Calibration` values from the enum? https://github.com/quantumlib/Cirq/blob/main/cirq-google/cirq_google/engine/result_type.py